apps/wallet/src/components/request-policies/rule/NamedRule.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="d-flex align-center justify-start">
+    {{ $t('request_policies.rule.named_rule') }}
+    <template v-if="loading">
+      <VProgressCircular indeterminate />
+    </template>
+    <template v-else-if="error">
+      {{ $t('request_policies.rule.named_rule') }}
```

Comment:
the error should be surfaced here instead of repeating the same thing right? or the namedRuleId